app/Controllers/Http/ProfilesController.ts

```diff
@@ -0,0 +1,143 @@
+import ProfileValidator from "App/Validators/ProfileValidator";
+import Profile from "App/Models/Profile";
+import User from "App/Models/User";
+import moment from "moment";
+
+class ProfilesController {
+  public async getProfile({ auth, response }) {
+    try {
+      const userData = await auth.user.id;
+      const profile = await Profile.findBy("user_id", userData);
```

Comment:
we can directly use `auth.user.id` in findBy method, why use another variable, and why to use `await` to access `id`